app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,33 @@
 class BookingsController < ApplicationController
+  def new
+    @booking = Booking.new
+    @user = User.find(params[:user_id])
```

Comment:
@user = current_user (courtesy of devise)?

Reply:
Good shout, will update